src/components/VisualTesterKeymap.vue

```diff
@@ -208,10 +250,10 @@ export default {
       }
       this.scrollToEnd();
     },
-    formatKeyEvent(ev, endTS) {
+    formatKeyEvent(ev, time) {
       let msg = [];
-      if (endTS) {
-        msg.push(`in ${(endTS - this.timing[ev.code]).toFixed(3)}ms`);
```

Comment:
why did you remove the precision limit?

Reply:
it has just been moved into a function: https://github.com/qmk/qmk_configurator/pull/466/files#diff-44b9ededf7579010954e5efecdb111eeR188